### PR TITLE
[WIP] Update Dependencies and make this work

### PR DESCRIPTION
**Pull request by hacdias**
- [x] Update Dependencies
- [x] Fix Small bugs
- [x] Fix Welcome Screen
- [x] Move from radium to react inline styling which is exactly the same (https://reactjs.org/docs/dom-elements.html#style)
- [x] Stateless views are now functional

### CODE REVIEW

**Comment by hacdias** [comment]
GREEN! @diasdavid

File: package.json [hunk 1]
```diff
@@ -4,50 +4,52 @@
...
     "react-dnd": "^2.1.2",
     "react-dnd-html5-backend": "^2.1.2",
-    "react-dom": "^15.1.0",
-    "react-widgets": "^3.2.0",
+    "react-dom": "^16.1.1",
+    "react-widgets": "^4.1.1",
     "winston": "^2.1.0"
   },
   "devDependencies": {
     "babel-core": "^6.3.21",
```
Comment (diasdavid) [inline package.json]: While you are at it, can you add `aegir` as a devDep and use its `aegir lint` to make sure the linting rules are the same as the remaining modules?

File: package.json [hunk 2]
```diff
@@ -4,50 +4,52 @@
   "description": "IPFS Native Application",
```
Comment (diasdavid) [inline package.json]: Let's take off the `1.0.0-alpha.1`, replace by `0.1.0` (that is a old pratice)

File: src/init.js [hunk 3]
```diff
@@ -43,17 +43,23 @@ function pollStats (ipfs) {
     })
     .then(next)
 
+  // TODO: solve?
   ipfs.id()
     .then((peer) => {
+      console.log(peer)
```
Comment (diasdavid) [inline src/init.js]: Unnecessary console.log

File: package.json (hunk 1 above)
Comment (hacdias) [reply package.json]: Should I add it to webui too?

File: package.json (hunk 2 above)
Comment (hacdias) [reply package.json]: Done!

File: package.json (hunk 1 above)
Comment (hacdias) [reply package.json]: Done done 😄

**Comment by hacdias** [comment]
@diasdavid changed made!

File: package.json [hunk 4]
```diff
@@ -53,8 +56,8 @@
   },
   "scripts": {
     "start": "electron src/index.js",
-    "test": "aegir lint",
-    "lint": "aegir lint",
+    "test": "eslint .",
```
Comment (diasdavid) [inline package.json]: We really need some tests here :)

**Comment by diasdavid** [comment]
@hacdias is drag and drop working?

**Comment by hacdias** [comment]
@diasdavid yup, it's working. It doesn't send any message telling so tho.

**Comment by hacdias** [comment]
I see we're using notifier to send notifications. Windows 10 API has changed in the last update and I think that's the reason why I'm not receiving any notification. Source: https://github.com/mikaelbr/node-notifier/issues/208.

**Pull request merged**